Sources/Fuzzilli/FuzzIL/Analyzer.swift

```diff
@@ -187,6 +190,11 @@ struct ContextAnalyzer: Analyzer {
         } else if instr.op is BeginClassDefinition {
             // We are no longer in the previous context
             contextStack.append([.classDefinition, .function])
+        } else if instr.op is BeginSwitch {
+            contextStack.append([context, .switchCase])
+        } else if instr.op is BeginSwitchCase {
```

Comment:
Is this case even necessary? At this point, the top context must be `[prevContext, .switchCase]`, and so this doesn't really do anything. I think it can just be removed

Reply:
I think you mean the `BeginSwitchCase` entry. Yeah that seems right, I'll remove it